src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.card__name {
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  text-decoration: none;
+  color: $main-text-color;
+  margin-top: 40px;
```

Comment:
The `margin-top` value in `.card__name` seems inconsistent with the design requirements. Please ensure that the margin-top value corresponds to the design. If it's intended, make sure to document the reason for this deviation in a comment.